### ISSUE TITLE
[Bug] [a11y]Menu Settings-Touch target area for "Bookmark","Share","Forward"&"Refresh" needs to be heightened

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#9660]
## Prerequisites
- Have Google Accessibility Scanner installed.
### Steps to reproduce
1. Open app and tap on the 3 dot menu.
2. Scan the app.
### Expected behavior
There are no suggestions to make any changes.
### Actual behavior
Menu Settings - Overlay button - Bookmark
- [548,2595][762,2677]
This item's height is 24dp. Consider making the height of this touch target 48dp or larger.

Menu Settings - Overlay button - Share
- [762,2595][976,2677]
This item's height is 24dp. Consider making the height of this touch target 48dp or larger.

Menu Settings - Overlay button - Forward
- [976,2595][1190,2677]
This item's height is 24dp. Consider making the height of this touch target 48dp or larger.

Menu Settings - Overlay button - Refresh
- [1190,2595][1404,2677]
This item's height is 24dp. Consider making the height of this touch target 48dp or larger.

### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Firefox Preview Nightly 4/3 (Build #20940609)

**Screens:**
**Menu Settings - Overlay button - Bookmark**
![Screenshot_20200403-122019_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78346894-618ff000-75a8-11ea-92a4-e66e109f3994.jpg)

**Menu Settings - Overlay button - Share**
![Screenshot_20200403-122116_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78346905-65bc0d80-75a8-11ea-98db-06b4fd3447c6.jpg)

**Menu Settings - Overlay button - Forward**
![Screenshot_20200403-122142_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78346923-694f9480-75a8-11ea-8942-101eabf33f03.jpg)

**Menu Settings - Overlay button - Refresh**
![Screenshot_20200403-122159_Accessibility Scanner](https://user-images.githubusercontent.com/49859863/78346943-6d7bb200-75a8-11ea-9d3e-21550fa4422a.jpg)

**Comment by asadotzler** [comment]
There's plenty of room here to increase the tap target without changing the layout of the menu. The icons are large enough and the width of the buttons are OK but the height could use to go to the top and bottom of the containing space.

**Comment by mcarare** [comment]
This should be fixed along with https://github.com/mozilla-mobile/fenix/issues/10705.

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build from 5/29 with:
* OnePlus 5T (Android 9):
<img src="https://user-images.githubusercontent.com/40694158/83255513-3cd78380-a1b9-11ea-8aae-acc05a6950a5.png" width="300" height="550" />

* Google Pixel (Android 10), and
* Sony Xperia Z5 Premium (Android 7.1.1):
<img src="https://user-images.githubusercontent.com/40694158/83255500-33e6b200-a1b9-11ea-8fd4-6ee453788d59.png" width="300" height="550" />
